catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/workspace/transformations/EmbeddedListMetacardsHandler.java

```diff
@@ -0,0 +1,112 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.catalog.ui.metacard.workspace.transformations;
+
+import static java.util.stream.Collectors.toList;
+
+import com.google.common.annotations.VisibleForTesting;
+import ddf.action.ActionRegistry;
+import ddf.catalog.data.Metacard;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+import org.codice.ddf.catalog.ui.metacard.workspace.ListMetacardImpl;
+import org.codice.ddf.catalog.ui.metacard.workspace.WorkspaceAttributes;
+import org.codice.ddf.catalog.ui.metacard.workspace.transformer.WorkspaceTransformer;
+import org.codice.ddf.configuration.SystemBaseUrl;
+
+public class EmbeddedListMetacardsHandler extends EmbeddedMetacardsHandler {
+
+  @VisibleForTesting static final String LIST_ACTION_PREFIX = "catalog.data.metacard.list";
+  @VisibleForTesting static final String ACTIONS_KEY = "actions";
+
+  private static final Set<String> EXTERNAL_LIST_ATTRIBUTES = Collections.singleton(ACTIONS_KEY);
+  private final ActionRegistry actionRegistry;
+
+  public EmbeddedListMetacardsHandler(ActionRegistry actionRegistry) {
+    super(WorkspaceAttributes.WORKSPACE_LISTS, ListMetacardImpl.TYPE);
+    this.actionRegistry = actionRegistry;
+  }
+
+  // Add "actions" key to list metacards.
```

Comment:
❓ Should these be javadoc?

Reply:
This was originally javadoc but @Variadicism said that would hide the javadoc of the method it overrides.

Reply:
Use `{@inheritDoc}`. https://docs.oracle.com/javase/6/docs/technotes/tools/solaris/javadoc.html#inheritingcomments

Reply:
Added.